glowing_waffles/notebooks/observe class.ipynb

```diff
@@ -0,0 +1,300 @@
+{
```

Comment:
<p>I was confused here too because it seemed like `Rc` should be in the list.</p>

 _Reply via <a href='https://app.reviewnb.com/glowing-waffle/glowing-waffles/pull/36/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>

Reply:
seems I had two statements checking if the filter existed in self.filters in the rm_filter method.